src/GovUk.Education.ExploreEducationStatistics.Data.Model.Tests/Repository/FootnoteRepositoryTests.cs

```diff
@@ -13,247 +15,105 @@ namespace GovUk.Education.ExploreEducationStatistics.Data.Model.Tests.Repository
 {
     public class FootnoteRepositoryTests
     {
+        private readonly DataFixture _fixture = new();
+        private readonly Generator<Release> _releaseGenerator;
+        private readonly Generator<Footnote> _footnoteGenerator;
+        private readonly Generator<ReleaseFootnote> _releaseFootnoteGenerator;
+        private readonly Generator<ReleaseSubject> _releaseSubjectGenerator;
+        private readonly Generator<Subject> _subjectsGenerator;
+        private readonly Generator<Filter> _filterGenerator;
+        private readonly Generator<FilterGroup> _filterGroupGenerator;
+        private readonly Generator<FilterItem> _filterItemGenerator;
+        private readonly Generator<IndicatorGroup> _indicatorGroupGenerator;
+        private readonly Generator<Indicator> _indicatorGenerator;
```

Comment:
Similar to `FootnoteService`, don't think assigning generators to properties are necessarily a good idea.

Reply:
Yup will change and see if there's any issues - assuming it'll be fine.

Reply:
Done ta